package.json

```
@@ -21,10 +21,8 @@
     "test:watch": "vitest",
     "test:e2e:dev": "start-server-and-test dev http://localhost:3000 \"npx cypress open\"",
     "prepare": "husky install",
-    "pretest:e2e:run": "npm run build",
-    "test:e2e:run": "cross-env PORT=8811 start-server-and-test start:mocks http://localhost:8811 \"npx cypress run\"",
     "typecheck": "tsc -b && tsc -b cypress",
-    "validate": "run-p test lint typecheck test:e2e:run",
+    "validate": "run-p test lint typecheck e2e",
```

@grantnorwood not sure I understand this change (might have been Seth). We don't have an `e2e` task. Mind if we revert?

```suggestion
    "validate": "run-p test lint typecheck test:e2e:run",
```

---

Ugh, me working on 3-4 branches at the same time 🤦   I reverted that.